debian/yaru-theme-gtk.install

```diff
@@ -1,13 +1,7 @@
 usr/share/gtksourceview-*/styles/Yaru*.xml
-usr/share/themes/Yaru/gtk-2.0/
-usr/share/themes/Yaru/gtk-3.0/
-usr/share/themes/Yaru/gtk-4.0/
-usr/share/themes/Yaru/metacity-1
-usr/share/themes/Yaru/xfwm4
-usr/share/themes/Yaru/index.theme
-usr/share/themes/Yaru-dark/gtk-2.0/
-usr/share/themes/Yaru-dark/gtk-3.0/
-usr/share/themes/Yaru-dark/gtk-4.0/
-usr/share/themes/Yaru-dark/metacity-1
-usr/share/themes/Yaru-dark/xfwm4
-usr/share/themes/Yaru-dark/index.theme
+usr/share/themes/Yaru*/gtk-2.0/
+usr/share/themes/Yaru*/gtk-3.0/
+usr/share/themes/Yaru*/gtk-4.0/
+usr/share/themes/Yaru*/metacity-1
+usr/share/themes/Yaru*/xfwm4
+usr/share/themes/Yaru*/index.theme
```

Comment:
After thinking about this, I think the `yaru-theme-gtk` should only handle default and dark variant.
Each other variant should have its own package.

@Muqtxdir can you make the modifications please?

Reply:
oh shoot, I'll do that 👍

Reply:
Thanks :)